### DISCUSSION TITLE
how to use NATS message ?

### DISCUSSION THREAD

**Discussion by icoco** [edgexfoundry/community#278] [Q&A] [answered]
as topic, from the online document, this [go-mod-messaging]( https://github.com/edgexfoundry/go-mod-messaging) support NATS,

but while run code , it seem does not work , it report error : `to enable NATS message bus options please build using the flag include_nats_messaging`

it should rebuild the NATS server or how to solve this problem ?

**Comment by cloudxxx8** [comment, marked as answer]
Here is the related docs: https://docs.edgexfoundry.org/3.1/microservices/general/messagebus/#nats_1
